plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -480,6 +480,10 @@ public static boolean confirmAction(@Nullable Shell shell, final String title, f
         return confirmAction(shell, title, question, DBIcon.STATUS_QUESTION);
     }
 
+    public static boolean confirmDangerousAction(@Nullable Shell shell, String title, String question) {
```

Comment:
What about adding `@NotNull`/`@Nullable` annotations for the `title` and `question` parameters?